components/global/Header/NavItemWithChildren.js

```diff
@@ -9,12 +9,15 @@ import internalLinkShape, { internalLinkInternalShape } from "@/shapes/link";
 export default function NavItemWithChildren({
   id,
   active,
+  parentActive,
   title,
   uri,
   childItems,
   onToggleClick,
   onEsc,
   theme,
+  baseClassName = "c-nav-list",
+  level = 2,
```

Comment:
Introduced sense of "what level am I in the nav(s)" in order to recursively reuse the NavItemWithChildren and Subnavigation components